pkg/sources/adapter/azureservicebussource/adapter.go

```diff
@@ -180,27 +188,32 @@ func entityPath(entityID *v1alpha1.AzureResourceID) string {
 	}
 }
 
-// namespaceFromEnvironment mimics the behaviour of eventhub.NewHubFromEnvironment
-// by returning a servicebus.NamespaceOption that is suitable for the
+// clientFromEnvironment mimics the behaviour of eventhub.NewHubFromEnvironment
+// return a azservicebus.Client that is suitable for the
```

Comment:
The comment is cut. I think you meant to finish the sentence above and start a new one.

```suggestion
// clientFromEnvironment mimics the behaviour of eventhub.NewHubFromEnvironment.
// It returns a azservicebus.Client that is suitable for the
```